example/cmd/device-simple/Attribution.txt

```diff
@@ -112,4 +112,10 @@ edgexfoundry/go-mod-secrets (Apache 2.0) https://github.com/edgexfoundry/go-mod-
 https://github.com/edgexfoundry/go-mod-secrets/blob/master/LICENSE
 
 edgexfoundry/go-mod-configuration (Apache 2.0) https://github.com/edgexfoundry/go-mod-configuration
-https://github.com/edgexfoundry/go-mod-configuration/blob/master/LICENSE
\ No newline at end of file
+https://github.com/edgexfoundry/go-mod-configuration/blob/master/LICENSE
+
+fxamacker/cbor (MIT) https://github.com/fxamacker/cbor/v2
+https://github.com/fxamacker/cbor/blob/master/LICENSE
```

Comment:
ugorji/go needs to be removed from this file

Reply:
for some reason the test script request this attribute ...
maybe we create another issue to check the test script and cleanup the Attribution.txt file ?

Reply:
ok, please create another issue.
This library should be used by github.com/edgexfoundry/go-mod-core-contracts v0.1.36, because we haven't upgraded to the latest contract.  We could deal with that after the contract upgrade

Reply:
that makes sense.